package.json

```diff
@@ -25,14 +25,23 @@
     "gatsby-plugin"
   ],
   "dependencies": {
+    "remark": "^13.0.0",
     "to-gatsby-remark-plugin": "^0.1.0",
     "unist-util-visit": "^2.0.1"
   },
   "devDependencies": {
+    "import-local": "^3.0.2",
+    "jest": "^26.6.3",
     "prettier": "1.19.1"
   },
   "prettier": {
     "singleQuote": true,
     "trailingComma": "es5"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "installConfig": {
+    "pnp": true
```

Comment:
I'm actually think we could just switch to `npm` and disable lock file, but let's do it in another PR.